OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -36,29 +36,30 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			BuildMusicTable();
 
 			Func<bool> noMusic = () => !musicPlaylist.IsMusicAvailable;
+			Func<bool> disableControls = () => !musicPlaylist.IsMusicAvailable || musicPlaylist.AmbientMusic;
```

Comment:
You can think of this as defining a function:
```
bool _DisableControls()
{
	 return !musicPlaylist.IsMusicAvailable || musicPlaylist.AmbientMusic;
}
```

which is then assigned to the `disableControls` variable.   Executing `disableControls()` is equivalent to running this implicit (anonymous, in programming lingo) function `_DisableControls()`.